_http/catalogue.http

```diff
@@ -1 +0,0 @@
-GET http://localhost:4200/api/3/action/package_search?start=0&rows=5
```

Comment:
Can you please revert these changes? I believe there was a rebase/merging issue with this branch.

Reply:
Done